### PR TITLE
use develop branch and fix clippy

### PR DESCRIPTION
**Pull request by jerry-yu**
use develop branch and fix clippy

### CODE REVIEW

File: cita-network/src/node_manager.rs [hunk 1]
```diff
@@ -286,6 +286,8 @@ impl NodesManager {
         }
     }
 
+    // clippy
+    #[allow(clippy::drop_copy, clippy::zero_ptr)]
```
Comment (kaikai1024) [inline cita-network/src/node_manager.rs]: Why not fix it?

Comment (jerry-yu) [reply cita-network/src/node_manager.rs]: for clippy

Comment (jerry-yu) [reply cita-network/src/node_manager.rs]: maybe you can try it first

Comment (jerry-yu) [reply cita-network/src/node_manager.rs]: it is up to lib's macro select!

Comment (jerry-yu) [reply cita-network/src/node_manager.rs]: it is due to use new crossbeam-channel and the reson for use crossbeam-channel,is upgrade cita-logger in bft

Comment (kaikai1024) [reply cita-network/src/node_manager.rs]: ok

**Pull request merged**